test/cli-report-parsing.test.js

```diff
@@ -0,0 +1,34 @@
+const expect = require('chai').expect;
+process.env.NODE_ENV = 'test';
+
+let cli;
+
+const version = require('../package.json').version;
+
+describe('CLI port number parsing', () => {
+    beforeEach(function () {
+        // delete require cache to use program instance for consecutive runs.
+        delete require.cache[require.resolve('./cli/mockCLI.js')];
```

Comment:
The comment says what the code does. We should add "why".

Also, maybe we can find a better way that busting cache for this. 🤔

Reply:
Actually, this is no longer needed, this was a solution to a problem we won't have anymore due to the removal of environment variables. This has now been removed!